.travis.yml

```diff
@@ -40,6 +40,10 @@ addons:
       - gcc-multilib
       - lib32gcc-6-dev
 
+before_script:
+  - wget http://releases.linaro.org/components/toolchain/binaries/6.3-2017.02/aarch64-linux-gnu/gcc-linaro-6.3.1-2017.02-x86_64_aarch64-linux-gnu.tar.xz
```

Comment:
@jgunthorpe 6 minutes is good enough and it is better to run for every pull request. Anyway, we are not merging pull request immediately. The travis limit is 50 minutes.